src/ReactPlaceholder.js

```diff
@@ -80,14 +83,28 @@ export default class ReactPlaceholder extends React.Component {
   }
 
   render() {
-    return this.state.ready ? this.props.children : this.getFiller();
+    const { ready, holdPlaceholder } = this.state;
```

Comment:
we already have `ready`, can't see the need for another variable that does the same thing